manifests/0000_90_cluster-kube-apiserver-operator_01_prometheusrole.yaml

```diff
@@ -0,0 +1,17 @@
+apiVersion: rbac.authorization.k8s.io/v1
+kind: Role
+metadata:
+  # TODO this should be a clusterrole
```

Comment:
@openshift/sig-auth @openshift/sig-monitoring can you two confirm.  This looks like a cluster role that needs to be bound locally.

Reply:
hmm, I might be missing something, but I don't see the necessity for a binding here, cc @squat @metalmatze